pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -373,11 +373,6 @@ func (h *NetworkUtilsHandler) StartDHCP(nic *VIF, serverAddr net.IP, bridgeInter
 			if err = DHCPv6Server(
 				nic.IPv6.IP,
 				bridgeInterfaceName,
-				nameservers,
-				nic.Routes,
```

Comment:
This is not mentioned in the commit message.

Reply:
done